src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -28,6 +28,7 @@ function SearchPageBottomTab() {
     const [selectionMode] = useOnyx(ONYXKEYS.MOBILE_SELECTION_MODE);
     const searchParams = activeCentralPaneRoute?.params as AuthScreensParamList[typeof SCREENS.SEARCH.CENTRAL_PANE];
     const parsedQuery = SearchUtils.buildSearchQueryJSON(searchParams?.q);
+    const searchName = searchParams?.name;
```

Comment:
This change misses covering the case where the search name equals the query name which introduce [this issue](https://github.com/Expensify/App/issues/50223) so we should get searchName based on search query